### ISSUE TITLE
Paper-textarea Styling Problem on Edge + IE

### ISSUE THREAD

**Issue by firstor** [first87/polymer-chat#20]
Styling `paper-textarea` doesn't seem to work properly. It works fine on Chrome and FF, but not in Edge (v14.14393) and IE (v11.0.14393.0).

#### Expected Behavior:
![image](https://user-images.githubusercontent.com/20905447/27560338-a6e69e9a-5a78-11e7-8e4e-6c77ad82f3b2.png)

#### Error Behavior in Edge & IE:
![image](https://user-images.githubusercontent.com/20905447/27560366-c39ab5c6-5a78-11e7-96dd-177d20f405da.png)

#### Key changes to style paper-textarea
`src/app/message/message-item.component.css`
```css
.message-input-field {
    --paper-input-container-input-color: var(--app-color-green); /* NOT working */
    --paper-input-container-underline: { /* NOT working */
        display: none;
    }
    --paper-input-container-underline-focus: { /* NOT working */
        display: none;
    }
    @apply --app-word-wrapping; /* seems working fine */
}
```

Need to resolve this asap.

**Comment by firstor** [comment]
Refer this comment: https://github.com/hotforfeature/origami/issues/37#issuecomment-311530360 to get some background of the issue. It works fine.

**Comment by firstor** [comment]
Seems fixed with PR #21
